packages/react-email/package.json

```diff
@@ -29,6 +33,7 @@
     "@manypkg/find-root": "2.2.1",
     "@octokit/rest": "19.0.7",
```

Comment:
I believe this dependency can be removed now that we don't download the client anymore 🙏